security-analysis/security-analysis-api/src/main/java/com/powsybl/security/results/BranchResult.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.results;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+/**
+ * provide electrical informations on a branch
```

Comment:
Small typo, `information` is invariable.
Be specific about the information: of which state? when is this class used?